examples/fscloud/main.tf

```diff
@@ -83,4 +83,21 @@ module "secrets_manager" {
       }]
     }
   ]
+  secrets = [{
+    secret_group_name = "${var.prefix}-secret-group"
```

Comment:
There are two major code paths, existing_secret_group true or false. The default is false, so for the test we might want to use the secrets-manager-secret-group to module directly to create a new group and then this submodule to put secrets in it, setting exsting_secret_group true.
Alternatively we should validate the code path in a different example that is part of the PR pipeline.

Reply:
`existing_secret_group ` true and false scenario is now being tested in complete example